index.js

```diff
@@ -1,14 +1,20 @@
 #!/usr/bin/env node
 import { Command } from 'commander';
 import fs from 'node:fs';
+import path from 'node:path';
 
 import { install } from './commands/install.js';
 import { uninstall } from './commands/uninstall.js';
 import { clean } from './commands/clean.js';
 
 import { readConfig } from './utils/config.js';
 
-const packageJson = JSON.parse(fs.readFileSync('./package.json', 'utf8'));
+const packageJsonPath = path.join(
+  path.dirname(new URL(import.meta.url).pathname),
```

Comment:
The linter isn't happy with this usage: `import.meta.url`, will iterate on this.